docs/pages/desktop-access/introduction.mdx

```diff
@@ -4,59 +4,70 @@ description: Demonstrates how you can manage access to Windows desktops with Tel
 videoBanner: n2h0GisWdss
 ---
 
-The topics in this guide describe how to configure Teleport to provide secure, passwordless 
+The topics in this guide describe how to configure Teleport to provide secure, passwordless
 access to Microsoft Windows desktops and servers. For Windows, Teleport provides the
 following key features:
 
-- Passwordless access to Windows hosts backed by secure cryptographic authentication.
+- Passwordless access to Windows hosts backed by secure cryptographic
+  authentication and short-lived certificates.
 - Configurable role-based access controls (RBAC) for groups of hosts and users.
-- Configurable clipboard and directory sharing for copying and pasting to and from 
-remote Windows hosts.
+- Configurable clipboard and directory sharing for copying and pasting to and
+  from remote Windows hosts.
 - Session recording for all desktop activity.
 - Audit logs that track user activity.
 
-Teleport Windows Desktop Services relies on the remote desktop protocol (RDP) to connect to
-remote Windows hosts. Therefore, Teleport only supports the following Windows hosts that are 
-accessible over RDP:
+## How it works
 
-- Windows Server 2012 R2, or later.
-- Windows 10, or later.
-
-The following diagram provides a simplified view of the architecture for managing
-access to Windows computers through Teleport:
+In order to provide connectivity to Windows hosts, you deploy one or more
+instances of Teleport's Windows Desktop Service on your infrastructure. The
+Windows Desktop Service typically makes an outbound connection to the Teleport
+Proxy Service and maintains a reverse tunnel that is used to power remote
+desktop sessions. Since the only networking requirement is a single outbound
+connection to the Teleport Proxy, you are free to deploy as many desktop
+services in as many different network segments as you need - Teleport will take
+care of selecting a route to your Windows host at connection time.
```

Comment:
> Since the only networking requirement is a single outbound connection to the Teleport Proxy

There's also a networking requirement of wds --> RDP server.